src/filebrowser/listing.ts

```diff
@@ -354,14 +354,10 @@ class DirListing extends Widget {
    */
   delete(): Promise<void> {
     let names: string[] = [];
-    if (this._softSelection) {
-      names.push(this._softSelection);
-    } else {
-      let items = this._model.items;
-      for (let item of items) {
-        if (this._selection[item.name]) {
-          names.push(item.name);
-        }
+    let items = this._model.items;
+    for (let item of items) {
+      if (this._selection[item.name]) {
+        names.push(item.name);
       }
     }
     let message = `Permanantly delete these ${names.length} files?`;
```

Comment:
`Permanantly` => `Permanently`

This message may be worded better altogether, I'm not sure.